supabase/functions/_backend/private/validate_password_compliance.ts

```
@@ -103,6 +104,8 @@ app.post('/', async (c) => {
 
   const userId = signInData.user.id
 
+  supabaseAdmin = useSupabaseAdmin(c)
```

Biome (2.1.2)

[error] Can't assign supabaseAdmin because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)